socorro/processor/breakpad_transform_rules.py

```diff
@@ -459,3 +490,153 @@ def _predicate(
         return 'create_dump_lookup' in raw_crash
 
 
+#==============================================================================
+class BreakpadStackwalkerRule2015(ExternalProcessRule):
+
+    required_config = Namespace()
+    required_config.add_option(
+        name='symbols_url',
+        doc='url of the symbol server',
+        default="https://localhost"
+    )
+    required_config.command_line = change_default(
+        ExternalProcessRule,
+        'command_line',
+        'timeout -s KILL 30 {command_pathname} '
+        '--raw-json {raw_crash_pathname} '
+        '--symbols-url {symbols_url} '
+        '--symbols-cache {symbol_cache_path} '
+        '{dump_file_pathname} '
+        '2>/dev/null'
+    )
+    required_config.command_pathname = change_default(
+        ExternalProcessRule,
+        'command_pathname',
+        '/data/socorro/stackwalk/bin/stackwalker',
+    )
+    required_config.add_option(
+        'symbol_cache_path',
+        doc='the path where the symbol cache is found (quote path with '
+        'embedded spaces)',
+        default='/mnt/socorro/symbols',
+    )
+    required_config.add_option(
+        'temporary_file_system_storage_path',
+        doc='a path where temporary files may be written',
+        default='/tmp',
```

Comment:
It's no biggie. If someone's system's temp dir is /var/cache/tmp instead, for example, when they run some `--admin.print_conf` command it'll become that instead of `'/tmp'` or the python `tempfile.gettempdir()`